draft-ietf-quic-transport.md

```diff
@@ -1626,14 +1626,14 @@ payloads of at least 1200 bytes, adding padding to packets in the datagram as
 necessary. Sending padded datagrams ensures that the server is not overly
 constrained by the amplification restriction.
 
-Packet loss, in particular loss of a Handshake packet from the server, can cause
-a situation in which the server cannot send when the client has no data to send
-and the anti-amplification limit is reached. In order to avoid this causing a
-handshake deadlock, clients MUST send a packet upon a probe timeout, as
-described in {{QUIC-RECOVERY}}. If the client has no data to retransmit and does
-not have Handshake keys, it MUST send an Initial packet in a UDP datagram of
-at least 1200 bytes.  If the client has Handshake keys, it SHOULD send a
-Handshake packet.
+Loss of an Initial or Handshake packet from the server can cause a situation in
+which the server cannot send due to the anti-amplification limit
+and the client has no data to send because client Initial data has been
+acknowledged and it does not yet have Handshake data to send. In order to avoid
```

Comment:
This long compound sentence probably needs to be split up.

> Loss of an Initial or Handshake packet from the server can cause a deadlock if the client does not send additional Initial packets.  The server can reaches its anti-amplification limit (see {{xxx}}), but the client has received and acknowledged Initial packets without having a need to send a Handshake packet.  In this case, where the client would otherwise not send any additional packets, the server will be unable to send because it has not received enough from the client.  To prevent this deadlock, clients MUST send a packet on a probe timeout, or PTO; see Section XX of {{QUIC-RECOVERY}}.  If [...]

Reply:
Thanks, I tried to rewrite based on your suggestion.  Tell me what you think.